lib/ffi/pointer.rb

```diff
@@ -31,17 +31,24 @@
 #
 
 require 'ffi/platform'
+
+# NOTE: all method definitions in this file are conditional on
+# whether they are not already defined. This is needed because
+# some Ruby implementations might already provide these methods
```

Comment:
Could you add that it means TruffleRuby specifically?

Reply:
Done